src/lib/notcurses.c

```diff
@@ -593,6 +593,16 @@ inline int ncplane_cursor_move_yx(ncplane* n, int y, int x){
   return 0;
 }
 
+inline int ncplane_cursor_move_rel(ncplane* n, int y, int x){
+  if (n->y + y == -1){
+    logerror("Invalid target y -1)\n");
+    return -1;
+  }else if (n->x + x == -1){
+    logerror("Invalid target x -1)\n");
```

Comment:
here too, the fields shake under the thundering hooves of runaway rparens

Reply:
> it would be nice to have unit tests on these, but it's simple enough that i'm fine without them. we're gonna absolutely need some on the VT code, though! =] i'll approve, but please kill the rparens

Will do!